sway/config.c

```diff
@@ -53,7 +54,7 @@ static void free_mode(struct sway_mode *mode) {
 	free(mode);
 }
 
-void free_config(struct sway_config *config) {
+void free_config(struct sway_config *config, struct sway_config *new_config) {
```

Comment:
Done the extra argument instead, not sure if you can access the old code somewhere as force pushed but alternatiive to that was renaming config to old_config or something to access the global variable.

Reply:
Don't have time to do a proper review but -1 to whatever's going on here

Reply:
Can do a separate pass instead of trying to force my way into accessing the new config here I guess, renaming this variable is just as bad in my opinion